driver-ui/src/main/java/eu/tsystems/mms/tic/testframework/execution/worker/finish/WebDriverShutDownWorker.java

```diff
@@ -62,9 +58,7 @@ public void onMethodEnd(MethodEndEvent methodEndEvent) {
                     }
 
                     if (close) {
-                        TesterraListener.getEventBus().post(new BeforeShutdownWebDriverSessionsEvent(methodEndEvent));
-                        WebDriverManager.shutdown();
-                        TesterraListener.getEventBus().post(new AfterShutdownWebDriverSessionsEvent(methodEndEvent));
+                        WebDriverManager.forceShutdown();
```

Comment:
There is a difference between `forceShutdown()` and `shutdown()`: `shutdown()` checks if a session close is allowed, `forceShutdown()` does not.
Please also look for `tt.on.state.testfailed.skip.shutdown` property.

Reply:
Ah... I forgot to note the `close` value...